src/createBot.test.ts

```diff
@@ -191,17 +193,24 @@ describe('send functions', () => {
   });
 });
 
+function log(line: string) {
+  fs.appendFileSync('./log.txt', `${line}\n`);
+}
+
 describe('server functions', () => {
   const bot = createBot(fromPhoneNumberId, accessToken, { version });
   let server: Server | undefined;
   let app: Application | undefined;
 
   beforeAll(async () => {
+    log('✔️ Server is Runing');
     ({ server, app } = await bot.startExpressServer({ webhookVerifyToken }));
+    log(JSON.stringify(server?.address()));
   });
 
   afterAll((): Promise<void> => new Promise((resolve) => {
     if (!server) {
+      log('❌ Server Stopped');
```

Comment:
The server is not really stopped; it doesn't exist. Perhaps we can rename this to "All tests done"? :)